app/src/test/java/com/mertceyhan/bitcoinmarket/core/ui/LayoutViewStateTest.kt

```diff
@@ -24,12 +24,12 @@ class LayoutViewStateTest {
 
     private val httpException = mockk<HttpException>()
 
-    private lateinit var layoutViewState: LayoutViewState
+    private lateinit var layoutViewState: com.mertceyhan.bitcoinmarket.base.ui.LayoutViewState
```

Comment:
Could you add it as import to the top of the file?

`import com.mertceyhan.bitcoinmarket.base.data.State`